frontend/src/components/Main/OptionsForm.js

```diff
@@ -0,0 +1,133 @@
+import React, {Component} from "react";
+import OptionsList from "./OptionsList";
+
+import {inject, observer} from "mobx-react";
+
+@inject("DataStore")
+@observer
+class OptionsForm extends Component {
```

Comment:
i'd break this up into two components. an `OptionsForm` should just have a single `option`, and then there should be an `OptionsFormArray` that wraps one or more `OptionsForm`